src/persistence/idpay.ts

```diff
@@ -521,3 +521,30 @@ range(0, walletConfig.suspendedCount).forEach(() => {
     transactionOperationDetail
   ];
 });
+
+range(0, walletConfig.discountCount).forEach(() => {
+  const initiativeName = `${faker.company.name()} [D]`;
+
+  const initiative: InitiativeDTO = {
+    ...generateRandomInitiativeDTO(),
+    initiativeName,
+    initiativeRewardType: InitiativeRewardTypeEnum.DISCOUNT,
+    status: InitiativeStatus.REFUNDABLE,
+    iban: undefined,
+    nInstr: 0
```

Comment:
```suggestion
    nInstr: 0,
    accrued:0,
    refunded:0
```
if I'm not mistaken, `DISCOUNT` initiatives get locked after the first transaction at the moment, so at least to me it makes sense to have them either "unspent" or "fully spent"; what do yo think?

Reply:
Fixed in https://github.com/pagopa/io-dev-api-server/pull/250/commits/920dae56da7a08df3ddaab222c44a43679c9c289